### ISSUE TITLE
Support Polygon for OpenSea

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#3622]
Eg. collection on Polygon: https://opensea.io/collection/darkdormneighbors

- [ ] Verify it works (i.e. it is supported)
- [ ] Create Android issue
- [ ] Implement for iOS

**Comment by oa-s** [comment]
@hboon  do we need to user specific url for `poligon` server like we do for `mainet` i mean that we use `https://api.opensea.io/ ` for  mainet;
`https://rinkeby-api.opensea.io/` for rinkeby;

**Comment by oa-s** [comment]
@hboon need your help of configuring infura project to enable poligon in infura console

**Comment by hboon** [comment]
> @hboon do we need to user specific url for `poligon` server like we do for `mainet` i mean that we use `https://api.opensea.io/ ` for mainet;
> `https://rinkeby-api.opensea.io/` for rinkeby;

I have no idea :)

**Comment by hboon** [comment]
> @hboon need your help of configuring infura project to enable poligon in infura console

If you are using da3717f25f824cc1baa32d812386d93f (this one and only this one is safe to paste here, because it's embedded in the repo), it should be enabled for Polygon already. Actually all our Infura keys should be enabled for Polygon.

**Comment by hboon** [comment]
> > @hboon do we need to user specific url for `poligon` server like we do for `mainet` i mean that we use `https://api.opensea.io/ ` for mainet;
> > `https://rinkeby-api.opensea.io/` for rinkeby;
> 
> I have no idea :)

I took a quick look at the docs before creating the issue and didn't see their docs mentioning the endpoint, so not sure how it works or if it is provided at all.

**Comment by oa-s** [comment]
ok, trying to find out appropriate account with tokens on `Poligon` for testing

**Comment by hboon** [comment]
Thanks. You probably know, but just in case this might save you some time, you can go to a collection on OpenSea eg https://opensea.io/collection/darkdormneighbors. Click one of the NFT, look for "Owned by" at the top and get an address.

**Comment by oa-s** [comment]
`0x0b380e76cb334b2949ed218b748e5417199aca69` - is owner of https://opensea.io/collection/darkdormneighbors
so i sent request kind of this `https://api.opensea.io/api/v1/assets/?owner=0x0b380e76cb334b2949ed218b748e5417199aca69&order_by=pk&order_direction=asc&limit=50&offset=0`
and as response there is an empty response:
```
HTTP 200 OK
Allow: GET, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "assets": []
}
```

**Comment by hboon** [comment]
Oh. So it isn't supported yet?

According to https://docs.opensea.io/reference/api-overview:

> Looking for Polygon APIs? Our V2 API is not quite ready yet, but stay tuned for more updates soon!

**Comment by hboon** [comment]
If you can't get anything out of it, let's close the PR for the time being and hold this first then.

**Comment by oa-s** [comment]
@hboon  that is strange but for some reasons the OpenSea API doesn't  return some tokens, especially those are on polygon

**Comment by oa-s** [comment]
@hboon  oh, out of their documentation polygon doesn't support by api https://docs.opensea.io/reference/api-overview
`Looking for Polygon APIs? Our V2 API is not quite ready yet, but stay tuned for more updates soon!`

**Comment by hboon** [comment]
Yeah. Let’s just leave this for now and check back again. I thought there’s a chance it already worked.
